### PR TITLE
CB-10996 Updating docs index page to match new content.

### PR DESCRIPTION
**Pull request by dblotsky**
@riknoll @nikhilkh @rakatyal @devgeeks please take a look when you have a moment.

### CODE REVIEW

**Comment by dblotsky** [comment]
Screenshot:
capture

**Comment by devgeeks** [comment]
Looks nice

**Comment by riknoll** [comment]
@dblotsky why change the headers from what's in the TOC? It might be a little weird to have two links on the page with different text going to the same place. Same with "Writing Apps/Plugins" vs. "Create Apps/Plugins". Also, why are some of the headings in the TOC  here and others not? The platform guides need entries in my opinion.

**Comment by dblotsky** [comment]
@riknoll please give this another look. It's now using the changes from [CB-11126][1].

[1]: https://issues.apache.org/jira/browse/CB-11126

File: www/docs/en/dev/guide/hybrid/webviews/index.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ license: >
 
 title: Embedding WebViews
 toc_title: Embed Cordova in native apps
+description: Implement the Cordova WebView in your native project.
```
Comment (riknoll) [inline www/docs/en/dev/guide/hybrid/webviews/index.md]: Include the cordova Webview?

File: www/docs/en/dev/guide/appdev/whitelist/index.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ license: >
 
 title: Whitelist Guide
 toc_title: Whitelisting
+description: Grant an application access to external resources.
```
Comment (riknoll) [inline www/docs/en/dev/guide/appdev/whitelist/index.md]: Securely grant an application... Just trying to emphasize that this is a security feature.

File: www/docs/en/dev/guide/support/index.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ license: >
 
 title: Corodva support by platform
 toc_title: Platform support
+description: Compatibility table for all major features.
```
Comment (riknoll) [inline www/docs/en/dev/guide/support/index.md]: Plugins and features

**Comment by riknoll** [comment]
LGTM! Just a few nitpicks on the descriptions themselves. Code looks good

**Pull request merged**